CHANGES

```diff
@@ -1,3 +1,6 @@
+blueox (0.11.6.2)
+  * Fix handling of non utf-8 characters
```

Comment:
This doesn't actually fix that - more accurate would be "fix handling of non-ascii characters" or "fix handling of unicode strings".

Reply:
stfu